metron-analytics/metron-maas-service/README.md

```diff
@@ -95,6 +95,12 @@ usage: ModelSubmission
  -zr,--zk_root <arg>             Zookeeper Root
 ```
 
+## Kerberos Support
+
+Model as a service will run on a kerberized cluster with a caveat.  The user who submits 
+the service will be the user who executes the models on the cluster.  That
+is to say that user impersonation of models deployed is not done at the moment.
+
 ##Stellar Integration
```

Comment:
I am definitely going to fix this here, but this is the case all through our docs suddenly.  I think github changed how they are rendering their markdown.  Just food for thought.